### PR TITLE
Update map tile 

### PR DESCRIPTION
**Pull request by RyanMarshall5765**
Adds ApiKey.js to store mapbox API key.
Updates tileURL and mapAttribution in Config.js to reflect the new tileset.
Adds dummy file with instructions for ApiKey.js 
Adds import in map for apikey and concatenated API key to map URL.

Not sure if I overloaded the map with layers. Seems to be an issue only in Firefox though from googling it may be on leaflets end.

Firefox issue: Will-change memory consumption is too high. Budget limit is the document surface area multiplied by 3 (1559040 px). Occurrences of will-change over the budget will be ignored.

Issue: #15

### CODE REVIEW

File: src/containers/Config.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ export const Config = {
   centerPoint: [37.73, 14.2],
   initialZoom: 8,
   maxZoom: 18,
-  tileURL: 'http://tile.stamen.com/terrain/{z}/{x}/{y}.jpg',
+  tileURL:
+    'https://api.mapbox.com/styles/v1/marshallr2/cjmgpleyx0lri2tph0orws1m5/tiles/256/{z}/{x}/{y}@2x?access_token=',
```
Comment (joephayes) [inline src/containers/Config.js]: if this URL is specific to you account then let's create a generic account for the project.

**Comment by joephayes** [comment]
Looks good overall - question about MapBox URL

File: src/containers/Config.js (hunk 1 above)
Comment (joephayes) [reply src/containers/Config.js]: I just confirmed - using this URL and my access token I get 404.

Comment (RyanMarshall5765) [reply src/containers/Config.js]: I made it public. If you could test to ensure that fixes the issue then we can move the map to an account name fitting the project.

Comment (joephayes) [reply src/containers/Config.js]: It works! Let's create the new account and update the URL

**Pull request merged**